smoketests/__init__.py

```diff
@@ -236,7 +236,11 @@ def stderr_task():
             sys.stderr.writelines(proc.stderr)
         threading.Thread(target=stderr_task).start()
 
-        print("initial update:", proc.stdout.readline())
+        init_update = proc.stdout.readline()
+        code = proc.poll()
+        if code:
+            raise subprocess.CalledProcessError(code, fake_args)
+        print("initial update:", init_update)
```

Comment:
Was giving me an annoying error because I had named the table in the new smoketest "Table", and I was so confused why `SELECT * FROM Table` was failing with a parse error (sql is case-insensitive and `TABLE` is a keyword). This was part of my debugging that - it makes `self.subscribe()` raise the error earlier instead of delaying it til you call `sub()`.

Reply:
FWIW this was removed in https://github.com/clockworklabs/SpacetimeDB/pull/1536